src/set_theory/zfc.lean

```diff
@@ -682,19 +682,25 @@ def Class := set Set
 
 namespace Class
 
+/-- The preferred way to convert a set of Sets into a Class. -/
+def of_set : set Set ≃ Class := equiv.refl _
```

Comment:
Do `set Set` and `Class` disagree on any instances?

Reply:
Yes, they disagree on set membership. Classes do t have an instance for `haa_mem Set Class`.